active-model-adapter.d.ts

```diff
@@ -1,22 +1,12 @@
-import Ember from 'ember';
```

Comment:
it seems that because the .d.ts files are checked in that git sees the file conversion as a move to the .d.ts files instead of as one to the .ts files. Let's remove the .d.ts files and keep only the .ts files.

Reply:
Done